packages/read-and-create-calendar-events/backend/node/README.md

```diff
@@ -1,47 +1,61 @@
-# Read and create calendar events
+# Read and create calendar events Nylas sample app
 
 An implementation with Node Express.
 
 ## Requirements
 
-- Node 18.0.0 or later
-- Configured .env file
+- Node 18.0.0 or later (see [checking the Node version](#checking-the-nodejs-version))
+- [a .env file with your Quickstart app secrets](#set-up-your-env-file)
 
-### Checking the Node.js version
+## Running this app independently
 
-To check which Node version you have, run the following command in your terminal:
+### Set up your `.env` file
+
+Go to the Nylas Dashboard, and choose the Quickstart Application.
+
+Click **App Settings** to see the `client_id` and `client_secret` for the Quickstart app.
```

Comment:
Hey @sanscontext there is a mention of CLIENT_URI in the code. Should this be added as well?

Reply:
I have no idea what that is, and I haven't encountered that in running these. @AaronDDM  help? :)

Reply:
I believe CLIENT_URI referrers to the URL of the frontend app.

@sanscontext i realized this text still calls the app a “QuickStart app” instead of “sandbox”

Reply:
> this text still calls the app a “QuickStart app” instead of “sandbox”

I think that's safe to leave until Sandbox is at 100% rollout? Happy to change it if needed.